internal/pkg/deploy/cloudformation/stack/lb_web_svc.go

```diff
@@ -20,6 +21,7 @@ const (
 	lbWebSvcRulePriorityGeneratorPath = "custom-resources/alb-rule-priority-generator.js"
 	desiredCountGeneratorPath         = "custom-resources/desired-count-delegation.js"
 	envControllerPath                 = "custom-resources/env-controller.js"
+	nlbCertManagerPath                = "custom-resources/nlb-cert-validator-updater.js"
```

Comment:
nit: maybe in a separate revision or PR we can rename the custom resource to 
```suggestion
	nlbCertManagerPath                = "custom-resources/nlb-cert-manager.js"
```